node/src/database/db_migrations/migrations/migration_9_to_10.rs

```diff
@@ -0,0 +1,71 @@
+use crate::database::db_migrations::db_migrator::DatabaseMigration;
+use crate::database::db_migrations::migrator_utils::DBMigDeclarator;
+
```

Comment:
Ohhhhhhh. This took me a good amount of time before I realised. That what I want to tell you is probably not going to be good news to you.

It will affect a lot of comments that I'd made before this moment.  

Thinking of schema of the database, you must admit that with your card there has been _no_ change. Not even a fingernail. 

The data types in the columns for the config table and the values of the parameters have ever been declared as an optional TEXT.

Which is exactly what you are using. 

Now, let's thing about what older databases could have had in their row for the start_block. Well, it was always a number. (And even if there was a way how it could yield 'Null', it would still be completely in accordance with your design.

That means _you don't need any db migration at all_. For that reason, you should make sure all the related code will go away. I'm sorry. 

Please don't forget delete this whole file.